src/components/navigation/tabs/z-navigation-tabs/index.tsx

```diff
@@ -146,6 +158,59 @@ export class ZNavigationTabs {
     });
   }
 
+  /**
+   * move focus though tabs using keyboad arrows.
+   */
+  @Listen("keydown")
+  private navigateThroughTabs(e: KeyboardEvent): void | boolean {
+    if (!this.isArrowNavigation(e)) {
+      return true;
+    }
+
+    if (this.isArrowNavigation(e)) {
+      e.preventDefault();
+      const children = Array.from(this.host.children);
+      children[this.tabFocus].querySelector('[role="tab"]').setAttribute("tabindex", "-1");
+      // Move forward
+      if (
+        (e.key === "ArrowRight" && this.orientation == NavigationTabsOrientation.HORIZONTAL) ||
+        (e.key === "ArrowDown" && this.orientation == NavigationTabsOrientation.VERTICAL)
+      ) {
+        this.tabFocus++;
+        if (this.tabFocus >= children.length) {
+          this.tabFocus = 0;
+        }
+        // Move backward
+      } else if (
+        (e.key === "ArrowLeft" && this.orientation == NavigationTabsOrientation.HORIZONTAL) ||
+        (e.key === "ArrowUp" && this.orientation == NavigationTabsOrientation.VERTICAL)
+      ) {
+        this.tabFocus--;
+        if (this.tabFocus < 0) {
+          this.tabFocus = children.length - 1;
+        }
+      }
+      //ignore disabled tabs
+      if (children[this.tabFocus].querySelector('[role="tab"]').hasAttribute("disabled")) {
+        this.navigateThroughTabs(e);
+      } else {
+        children[this.tabFocus].querySelector('[role="tab"]').setAttribute("tabindex", "0");
+        (children[this.tabFocus].querySelector('[role="tab"]') as HTMLElement).focus();
+      }
+    }
+  }
+
+  /**
+   * move focus though tabs using keyboad arrows.
+   */
+  private isArrowNavigation(e: KeyboardEvent): boolean {
+    return e.key === "ArrowRight" || e.key === "ArrowLeft" || e.key === "ArrowUp" || e.key === "ArrowDown";
```

Comment:
Sarebbe carino crearsi un enum con i 4 possibili valori, come è già stato fatto con NavigationTabsOrientation

Reply:
Qui il check poi dovrebbe diventare una cosa tipo `e.key in ArrowNavigationEnum`, ammettendo di aver chiamato così l'enum.